pkg/frontend/installversions_get.go

```diff
@@ -0,0 +1,33 @@
+package frontend
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/Azure/ARO-RP/pkg/frontend/middleware"
+	"github.com/Azure/ARO-RP/pkg/util/version"
+)
+
+func (f *frontend) getAvailableInstallVersions(w http.ResponseWriter, r *http.Request) {
+	log := r.Context().Value(middleware.ContextKeyLog).(*logrus.Entry)
+	versions := f.getInstallVersions()
+	b, err := json.Marshal(versions)
+	reply(log, w, nil, b, err)
+}
+
+// Creating this separate method so that it can be reused while doing some other stuff like validating the install version against available version.
+func (f *frontend) getInstallVersions() []string {
+	// TODO: Currently, the versions are hard coded and being pulled from version.UpgradeStreams, but in future they will be pulled from cosmosdb.
+	installStreams := version.UpgradeStreams
+	versions := make([]string, 0)
+	for _, installStream := range installStreams {
+		versions = append(versions, installStream.Version.String())
+	}
```

Comment:
We only support the installation of 4.10.15, so this function must only return the version defined in `version.InstallStream`:
https://github.com/Azure/ARO-RP/blob/4dd364aac52f1604285f42098869576a456812bd/pkg/util/version/const.go#L29-L32

Reply:
This would be still be temporary right ?

Reply:
So basically for now only 1 version that is the installStream. Got it. Changes done.